docs/en/sql-reference/functions/ip-address-functions.md

```diff
@@ -276,85 +292,116 @@ SELECT
 └───────────────────────────────────┴──────────────────────────┘
 ```
 
-## toIPv4OrDefault(string)
+## toIPv4OrDefault
 
-Same as `toIPv4`, but if the IPv4 address has an invalid format, it returns `0.0.0.0` (0 IPv4).
+Same as `toIPv4`, but if the IPv4 address has an invalid format, it returns `0.0.0.0` (0 IPv4), or the provided IPv4 default.
 
 **Syntax**
 
 ```sql
-toIPv4OrDefault(value)
+toIPv4OrDefault(string [, default])
 ```
 
 **Arguments**
 
-- `value` — A string-encoded IPv4 address. [String](../data-types/string.md)
+- `string` — IP address. [String](../data-types/string.md).
+- `default` (optional) — The default to return if `string` has an invalid format. [IPv4](../data-types/ipv4.md).
 
 **Returned value**
 
-- `value` converted to an IPv4 address. [IPv4](../data-types/ipv4.md).
+- `string` converted to the current IPv4 address. [String](../data-types/string.md).
 
 **Example**
 
 Query:
 
 ```sql
+WITH
+    '::ffff:127.0.0.1' AS valid_IPv6_string,
```

Comment:
It is too confusing to use *IPv6* addresses with *prefix "valid"* as examples for invalid IPv4 addresses. Please invent some invalid IPv4 addresses, e.g. `482.994.284.885` or `not.an.ip`.